src/main.rs

```diff
@@ -206,12 +206,7 @@ async fn run_upload(
     });
     let meta = BundleMeta {
         version: META_VERSION.to_string(),
-        cli_version: format!(
```

Comment:
should we just split this into 3 fields?

Reply:
Not sure why we had this in the first place, we don't use any of the other fields in the ETL it seems (we weren't even using `cli_version`). I think we log all 3 fields when running the CLI so might've just copied over that string